app/controllers/finance/banding_tracker/providers_controller.rb

```diff
@@ -21,7 +21,9 @@ def paid_participant_count_per_bands
       end
 
       def payable_participant_count_per_bands
-        @payable_participant_count_per_bands ||= lead_provider.participant_declarations.payable
+        @payable_participant_count_per_bands ||= lead_provider
```

Comment:
while we're here can we also code in the current cohort to prevent any panic attacks?